transforms/universal/fdedup/python/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM docker.io/python:3.10.14-slim-bullseye
+
+RUN pip install --upgrade --no-cache-dir pip 
+
+# install pytest
+RUN pip install --no-cache-dir pytest
+
+# Create a user and use it to run the transform
+RUN useradd -ms /bin/bash dpk
+USER dpk
+WORKDIR /home/dpk
+
+# Copy and install data processing libraries 
+# These are expected to be placed in the docker context before this is run (see the make image).
+COPY --chown=dpk:root data-processing-lib-python/ data-processing-lib-python/
```

Comment:
The new makefile will copy the whl file (not the source) to the context folder

Reply:
Fixed in commit https://github.com/IBM/data-prep-kit/pull/699/commits/6cc18cd8eaba2fb12a31f49af52aba188a9f6ac4